config/development.toml

```diff
@@ -229,3 +229,8 @@ checkout = { long_lived_token = false, payment_method = "wallet" }
 
 [connector_customer]
 connector_list = "stripe"
+
+[dummy_connector]
+payment_ttl = 172800
+payment_duration = 1000
+payment_tolerance = 100
```

Comment:
@ThisIsMani  Add these variables in config.example.toml & docker_compose.toml as well.
Can document these in config.example.toml